src/scss/src/atoms/Button.scss

```diff
@@ -0,0 +1,133 @@
+@import '../foundation/mixins';
+@import '../foundation/colors';
+@import '../foundation/typography';
```

Comment:
We do have file `_all.scss` isn't it better to import it here instead of 3 separate imports?

Reply:
Thanks for useful remark. Fixed